Source/OxyPlot/Series/LineSeries.cs

```diff
@@ -410,7 +410,7 @@ protected internal override void UpdateMaxMin()
                 // Make sure the smooth points are re-evaluated.
                 this.ResetSmoothedPoints();
 
-                if (this.SmoothedPoints.Count == 0)
+                if (this.SmoothedPoints.Where(x => !double.IsNaN(x.X)).Count() == 0)
```

Comment:
I'd prefer `.Any(x => !double.IsNaN(x.X))`: bit easier to see what's going on, and will be faster in the usual case where there are values.